cmd/ipfs/daemon.go

```diff
@@ -261,17 +247,18 @@ func daemonFunc(req cmds.Request, res cmds.Response) {
 	}
 
 	// construct fuse mountpoints - if the user provided the --mount flag
-	mount, _, err := req.Option(mountKwd).Bool()
-	if err != nil {
+	if err := mountFuse(req); err != nil {
 		res.SetError(err, cmds.ErrNormal)
 		return
 	}
-	if mount {
-		if err := mountFuse(req); err != nil {
+
+	// repo blockstore GC - if --disable-gc flag is not present
+	go func() {
+		if err := runGC(req, repo, node); err != nil {
 			res.SetError(err, cmds.ErrNormal)
 			return
 		}
-	}
+	}()
```

Comment:
i think this error should be delivered the same way that `apiErr, gwErrc` and so on are delivered. 

Though, i think even these are wrong. a failure in one of those will obscure all other failures. maybe we want something like:

```go
var errs []error
for err := range merge(apiErrs, gwErrs, mountErrs, gcErrs) {
  if err != nil {
    log.Error(err)
    errs = append(errs, err)
  }
}
if len(errs) > 0 {
  res.SerError(errs, cmds.ErrNormal)
  return
}
```
but also not 100% sure here. this stuff is tricky and not clean...

it may be ok to go with _almost_ what you have:


```go
gcErrc := make(chan error, 1)
go func() {
  defer close(gcErrc)
   if err := runGC(req, repo, node); err != nil {
      gcErrc<- errs
      return
    }
}()

...

merge(..., gcErrc)
```